src/Internal/CacheProvider.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tebru\Gson\Internal;
+
+use Psr\SimpleCache\CacheInterface;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\Cache\Adapter\ChainAdapter;
+use Symfony\Component\Cache\Adapter\NullAdapter;
+use Symfony\Component\Cache\Adapter\PhpFilesAdapter;
+use Symfony\Component\Cache\Adapter\Psr16Adapter;
+use Symfony\Component\Cache\Psr16Cache;
+use Symfony\Component\Cache\Simple\ArrayCache;
+use Symfony\Component\Cache\Simple\ChainCache;
+use Symfony\Component\Cache\Simple\NullCache;
+use Symfony\Component\Cache\Simple\PhpFilesCache;
+
+/**
+ * @codeCoverageIgnore
+ */
```

Comment:
Is this still necessary with the travis env variables?

Reply:
I suppose it is, nevermind.

Reply:
I make a commit to test... But it depends on Scrutinizer config, not travis

Reply:
Lost 1% in covering